pkg/antctl/runtime/runtime.go

```diff
@@ -47,10 +48,24 @@ func ResolveKubeconfig(path string) (*rest.Config, error) {
 			path = clientcmd.RecommendedHomeFile
 		}
 	}
-	if _, err = os.Stat(path); path == clientcmd.RecommendedHomeFile && os.IsNotExist(err) {
-		return rest.InClusterConfig()
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
I feel like the function could be cleaned up further. Maybe something like:
```go
withExplicitPath := path != ""
if !withExplicitPath {
        path = strings.TrimSpace(os.Getenv("KUBECONFIG"))
        if path == "" {
                path = clientcmd.RecommendedHomeFile // default path
        }
}
if _, err := os.Stat(path); err != nil {
        if !os.IsNotExist(err) {
                return nil, err // unexpected error
        }
        if withExplicitPath {
                // user-provided path is not valid, return error
                // ...
        }
        // no file found at default path, fallback to InClusterConfig
        // ...
}
```

Reply:
Yes, I feel this is better, updated.